src/Client.cpp

```diff
@@ -673,6 +682,11 @@ ::doHandler()
 
 		cleanupAfterLoop();
 	}
+#ifdef USING_VMA_EXTRA_API
```

Comment:
in Sockperf we don't care for memory leaks that happen just before exit and we prefer readability.  At any case, this is O(1) leak